lists2safebrowsing.py

```diff
@@ -650,7 +654,13 @@ def start_versioning(config, chunknum, shavar_prod_lists_branches):
     for branch in shavar_prod_lists_branches:
         branch_name = branch.get('name')
         ver = p_version.parse(branch_name)
+
         if isinstance(ver, p_version.Version):
+            # Delete records for a version that is no longer supported
+            if shared_state.latest_supported_version - ver.release[0] > int(config.get('main', 'num_supported_versions')):
+                deleteRecordsFromVersion(ver.release[0])
+                continue
+
```

Comment:
Why do we have to delete the unsupported remote setting versions here? Could we move this logic to `publish2cloud.publish_to_remote_settings()`?

Reply:
I've moved it to be done in that function now, it makes more sense